src/frontend/Sider/Behandling/VedtakOgBeregning/Læremidler/InnvilgeVedtak/InnvilgeLæremidler.tsx

```diff
@@ -63,7 +69,10 @@ export const InnvilgeLæremidler: React.FC<{
             `/api/sak/vedtak/laremidler/${behandling.id}/beregn`,
             'POST',
             vedtaksperioder
-        ).then(settBeregningsresultat);
+        ).then((result) => {
+            settBeregningsresultat(result);
+            if (result.status === 'SUKSESS') settErVedtaksperioderBeregnet(true);
```

Comment:
Kan være fint med curly brackets her for bedre lesbarhet
```suggestion
            if (result.status === 'SUKSESS') {
               settErVedtaksperioderBeregnet(true)
               };
```

Reply:
fikset